ddtrace/span.py

```diff
@@ -181,6 +184,31 @@ def set_metrics(self, metrics):
     def get_metric(self, key):
         return self.metrics.get(key)
 
+    def set_sampling_priority(self, sampling_priority):
+        """
+        Set the sampling priority.
+
+        0 means that the trace can be dropped, any higher value indicates the
+        importance of the trace to the backend sampler.
+        Default is None, the priority mechanism is disabled.
+        """
+        if sampling_priority is None:
+            self.priority = None
+        else:
+            try:
+                self.priority = int(sampling_priority)
+            except ValueError:
+                # if the provided sampling_priority is invalid, ignore it.
+                pass
+
+    def get_sampling_priority(self):
```

Comment:
Probably we need to change that. The pythonic way is to have a property instead of getter and setter (otherwise we'll finish having getter and setter for everything). Low priority, we can refactor that before the merge.